tests/playwright/sanity/bg-lazyload.test.js

```diff
@@ -1,47 +1,67 @@
 const { test, expect } = require( '@playwright/test' );
+const { createPage, deletePage } = require( '../utilities/rest-api' );
 const WpAdminPage = require( '../pages/wp-admin-page.js' );
+const EditorPage = require( '../pages/editor-page' );
 
-test.beforeEach( async ( { page }, testInfo ) => {
-	const wpAdmin = new WpAdminPage( page, testInfo );
-	await wpAdmin.setExperiments( {
-		e_lazyload: true,
+test.describe( 'Background Lazy Load', () => {
+	let pageId;
+
+	test.beforeAll( async ( { browser }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			e_lazyload: true,
+		} );
 	} );
-} );
 
-test( 'Background lazy load sanity test', async ( { page } ) => {
-	await page.goto( `/law-firm-about` );
-	// Get the first lezyload background element.
+	test.afterAll( async ( { browser }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			e_lazyload: false,
+		} );
+	} );
 
-	const lazyloadSelector = '[data-e-bg-lazyload]:not(.lazyloaded)';
-	await page.waitForSelector( lazyloadSelector );
-	const beforeURL = await page.$eval( lazyloadSelector, ( el ) => {
-		return window.getComputedStyle( el ).getPropertyValue( 'background-image' );
+	test.beforeEach( async () => {
+		pageId = await createPage();
 	} );
-	expect( beforeURL ).toContain( 'none' );
 
-	await page.evaluate( ( lazyloadSelectorScrollTo ) => {
-		const lazyloadElement = document.querySelector( lazyloadSelectorScrollTo );
-		lazyloadElement.scrollIntoView();
-		lazyloadElement.setAttribute( 'bg-test-element', '' );
-	}, lazyloadSelector );
+	test.afterEach( async () => {
+		await deletePage( pageId );
+	} );
 
-	await page.waitForTimeout( 1500 );
+	test( 'Background lazy load sanity test', async ( { context, page } ) => {
+		const editorPage = new EditorPage( page, { config: { projects: [ { use: {} } ] } } );
```

Comment:
```suggestion
		const editorPage = new EditorPage( page );
```

 I think we should remove the use of `testInfo` inside EditorPage, if you don't want to do so in this PR just use `testInfo`

```suggestion
		const editorPage = new EditorPage( page, testInfo );
```

Reply:
Fixed in https://github.com/elementor/elementor/pull/21063/commits/3ccf1d714be4a48cf2fc2db219fb0b33b034c384